homeassistant/components/verisure/config_flow.py

```diff
@@ -136,9 +137,16 @@ async def async_step_installation(
         self, user_input: dict[str, Any] | None = None
     ) -> FlowResult:
         """Select Verisure installation to add."""
+        installations_data = await self.hass.async_add_executor_job(
+            self.verisure.get_installations
+        )
         installations = {
-            inst["giid"]: f"{inst['alias']} ({inst['street']})"
-            for inst in self.verisure.installations or []
+            inst["giid"]: f"{inst['alias']}"
```

Comment:
Not sure why the address (street) was removed here?

Reply:
I didn't think it added any value, alias and street are the same, except the house number is missing from the alias.

Reply:
There are maybe the same for your system, but here they are not. For me, the alias is "Home", so it is the one from my parents.